x-pack/plugins/reporting/index.js

```diff
@@ -125,6 +127,7 @@ export const reporting = (kibana) => {
           }).default()
         }).default(),
         csv: Joi.object({
+          enablePanelActionDownlad: Joi.boolean().default(false),
```

Comment:
`enablePanelActionDownload` -- missing the `o`

Reply:
Wow, thanks!